archetypes/archetypes/src/main/archetype/mp/common/files/.idea/runConfigurations/configuration.xml.mustache

```diff
@@ -0,0 +1,9 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="{{artifactId}}" type="Application" factoryName="Application">
+    <option name="MAIN_CLASS_NAME" value="io.helidon.Main" />
```

Comment:
The class name shouldn't be hard-coded in this file. E.g. what if jpms is enabled ?